api.yaml

```diff
@@ -3506,6 +3506,34 @@ components:
           allOf:
             - $ref: "#/components/schemas/OptionsSymbol"
           nullable: true
+    OptionBrokerageSymbol:
+      description: Option Brokerage symbol
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/Id"
+        description:
+          type: string
+          example: SPY CALL 7/17 200
+        option_symbol:
+          allOf:
+            - $ref: "#/components/schemas/OptionsSymbol"
+          nullable: true
+        security_type:
+          $ref: "#/components/schemas/SecurityType"
+        listing_exchange:
```

Comment:
I think we should remove this field from the spec, I think we deprecated some time ago, looks like today we just return {} which isn't helpful